Library/Homebrew/test/formula_installer_bottle_spec.rb

```diff
@@ -23,6 +23,7 @@ def temporarily_install_bottle(formula)
     stub_formula_loader formula("gcc") { url "gcc-1.0" }
     stub_formula_loader formula("patchelf") { url "patchelf-1.0" }
     allow(Formula["patchelf"]).to receive(:latest_version_installed?).and_return(true)
+    described_class.new(formula).fetch
     described_class.new(formula).install
```

Comment:
You should be able to remove the line above and figure out why `fetch` isn't being called internally.